docs/SUMMARY.md

```diff
@@ -1,66 +1,72 @@
-<!-- markdownlint-disable -->
-# Summary
+# Quilt Documentation
 
-* [Introduction](README.md)
-* [Installation](Installation.md)
-* [Quick start](Quickstart.md)
-* [Mental model](MentalModel.md)
-* [Architecture](Architecture.md)
+* [About Quilt](README.md)
+* [Architecture](Architecture.md)  
```

Comment:
You've got a few trailing spaces in here and below that are not needed.

Reply:
Doh! I didn't realize markdown lint didn't warn about double-spaces at the end. Thanks.

Reply:
Also fixed the broken links, thanks.  Somehow, the file extensions were lost in translation.

Reply:
FYI, I tested all the links from the branch, and only found three that were dead.
https://github.com/quiltdata/quilt/blob/doc-reorg/docs/SUMMARY.md

Did I miss something?

Reply:
Triple-checked links, look good.